app/src/main/java/com/lagradost/cloudstream3/ui/result/EpisodeAdapter.kt

```diff
@@ -292,6 +297,12 @@ class EpisodeAdapter(
                     episodeDate.isVisible = false
                 }
 
+                episodeRuntime.setText(
+                    txt(
+                        card.runTime?.times(60L)?.let { secondsToReadable(it.toInt(), "0 mins") }
```

Comment:
0 mins? It should say 1 min or nothing at all.

Reply:
Any chance that this one will every be triggered?